lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -139,9 +142,10 @@ def process_dataset(child, full:, max_archive_size: DEFAULT_MAX_ARCHIVE_SIZE)
                 end
             end
 
-            def process_dataset_transfer(child, server, root, full:)
+            def process_dataset_transfer(child, server, root, full:, thresh:)
```

Comment:
I believe thresh is a bit undescriptive, I would call it min_required_space(and standarize this variable in the cli, instead of having multiple names for it)